internal/schedule_client.go

```diff
@@ -54,25 +54,38 @@ type (
 	// that means all years match. For all fields besides year, at least one Range must be present to match anything.
```

Comment:
this comment isn't accurate anymore?

Reply:
It is correct since at least one range for each field must be present still. We just have a default now. If you have a better wording I welcome any suggestions.